### PR TITLE
Add Cash text/button gradient

### PR DESCRIPTION
**Pull request by osdnk**
After: 
image

Before: 
image

### CODE REVIEW

File: src/components/AddFundsInterstitial.js [hunk 1]
```diff
@@ -96,14 +86,17 @@ const Subtitle = styled(Title)`
...
-  align: 'center',
-  children: android ? ` ${children} ` : children,
-  letterSpacing: 'roundedTightest',
-  size: 'bigger',
-  weight: 'heavy',
-}))`
-  ${android ? padding(15, 6) : padding(24, 15, 25)};
+const AmountText = styled(Text).attrs(
+  ({ children }) =>
+    console.log(children.join('')) || {
```
Comment (skylarbarrera) [inline src/components/AddFundsInterstitial.js:91]: 👁️

Comment (osdnk) [reply src/components/AddFundsInterstitial.js]: sorry :|.

File: src/components/AddFundsInterstitial.js [hunk 2]
```diff
@@ -86,13 +86,14 @@ const Subtitle = styled(Title)`
   margin-top: ${({ isSmallPhone }) => (isSmallPhone ? 19 : 42)};
 `;
 
-const AmountText = styled(Text).attrs({
+const AmountText = styled(Text).attrs(({ children }) => ({
   align: 'center',
+  children: android ? ` ${children.join('')} ` : children,
```
Comment (brunobar79) [inline src/components/AddFundsInterstitial.js:91]: Is there a reason for the leading & trailing spaces?

**Review by brunobar79 — APPROVED** [review]

File: src/components/AddFundsInterstitial.js (hunk 2 above)
Comment (brunobar79) [reply src/components/AddFundsInterstitial.js:91]: Oh I see... That's the hack for the shadows to not get cut off

**Review by brunobar79 — APPROVED** [review]

File: src/components/AddFundsInterstitial.js (hunk 2 above)
Comment (osdnk) [reply src/components/AddFundsInterstitial.js:91]: yes,

**Pull request merged**